net/ip/lwip_base/src/core/ipv6/nd6.c

```diff
@@ -703,6 +703,7 @@ nd6_input(struct pbuf *p, struct netif *inp)
 }
 
 
+#if LWIP_ND6_TMR 
```

Comment:
I don't see this define set/declared anywhere. Is this check supposed to be here?